apps/website/src/pages/docs/carousel.mdx

```diff
@@ -61,14 +61,18 @@ This component uses a composition approach so it should be used with the provide
 
 ### Navigation
 
-The `Carousel.Navigation` component by default consists of the `PreviousButton` and `NextButton` shown on the left and right, and the `Carousel.DotsList` component shown in the middle. `children` can be specified to override what is shown in this navigation section.
+The `Carousel.Navigation` component by default consists of the `PreviousButton` and `NextButton` shown on the left and right and the `Carousel.DotsList` component shown in the middle. `children` can be specified to override what is shown in this navigation section.
+
+**Important**: The navigation section must always be present before the slides in DOM order.
 
 <PropsTable path={'@itwin/itwinui-react/esm/core/Carousel/CarouselNavigation.d.ts'} />
 
 ### Dots List
 
 The `Carousel.DotsList` component shows a list of `Carousel.Dot` components which can be used to choose a specific slide. If used as a descendant of `Carousel`, then this component does not need any props or `children`. The props can be specified if this component is being used outside `Carousel`. `children` can be specified to override the individual dots that are shown.
 
+**Important**: The component must always be present before the slides in DOM order.
```

Comment:
since this component is optional and can also be used standalone, i believe we should clarify this further

Reply:
Suggested wording:

> **Important**: When used standalone, this component must always be present before the slides in DOM order.